lib/components/PoolStats.jsx

```diff
@@ -39,13 +39,7 @@ export const PoolStats = () => (
   </PoolStatsCard>
 )
 
-export const PoolDepositorStats = () => (
```

Comment:
I separated these lists so the landing page wasn't overloaded with info that's in the full list (time again, exit fees, reserve details). No objection to changing it to this, just letting you know.